RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# How to release edtr-io
+
+## Updating all packages:
+
+1. `yarn build` - run it locally to check if the build will pass
+2. `yarn regenerate-api` - run it locally to check if the generated API documentation is up to date
+3. `yarn prepare-release` - updates the Lerna version (which will be read by the GitHub deploy workflow (`/.github/workflows/deploy.yml`) once the release commit is pushed)
+4. Add a changelog entry in `/scripts/changelog.ts` and then run `yarn changelog` to generate a new changelog entry in `CHANGELOG.md`
+5. `git commit -m "release X.X.X"` - commit the changes with the correct release tag
```

Comment:
Unclear, if you need to set a git tag on this commit as well in addition to the commit message.

Reply:
Unclear, if all packages will have the same version number.

Reply:
> Unclear, if you need to set a git tag on this commit as well in addition to the commit message.

I changed the wording from `tag` to `version`, so it's not confused with a git tag.

> Unclear, if all packages will have the same version number.

This is something we will have to explore further. When we published 3.0.0, that's what happened. But I see that previous versions could increment only the packages that were changed. So we need to figure out why. My guesses are that it might have to do with the 3.0.0 being a major version, or maybe our changelog entry was not specific enough.

Reply:
Interesting to know. Maybe also add this observation to the documentation?